### ISSUE TITLE
AuthZ Extension Switching to Auth0 Core Losing Scope

### ISSUE THREAD

**Issue by jwstrivr** [auth0-samples/auth0-WinFormsWPF-oidc-samples#35]
### Checklist

- [X] I have looked into the Readme ([WPF](https://github.com/auth0-samples/auth0-WinFormsWPF-oidc-samples/tree/master/Quickstart/00-Starter-Seed/WPF#readme)/[WinForms](https://github.com/auth0-samples/auth0-WinFormsWPF-oidc-samples/tree/master/Quickstart/00-Starter-Seed/WinForms#readme)) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0-samples/auth0-WinFormsWPF-oidc-samples/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

With the upcoming deprecation of AuthZ Extension, we're looking to move our WPF Application over to using Auth0 Core/Groups. I've followed along with [documentation](https://auth0.com/docs/quickstart/native/wpf-winforms) with a completely barebones, stand-alone WPF app that just contains the `Auth0.Oidc.WPF ` nuget client and I'm able to get a successful authentication of my access token and returned user token with `Individual` login option, but the moment I switch to `Organization` the scope seems to be completely missing and I get an unauthenticated error return code (`openId profile email` are all missing in the logs when I switch to orgs).

### Reproduction

Here's a general overview of how I'm setting up the client:

```
            var auth0Client = new Auth0Client(new Auth0ClientOptions
            {
                Domain = "{my_domain}",
                ClientId = "{my_clientId}",
                Scope = "openid email profile",
            });
            loginResult = await auth0Client.LoginAsync(new { audience = "{my_audience}" });
```

Again, this works fine for individual login, but fails the moment I switch to orgs, which leaves me scratching my head. I am using the most up-to-date nuget package available, and only have these lines for my entire test app. Any help would be appreciated, thank you!

**Comment by frederikprijck** [comment]
Can you share the exact error that you retrieve as well as when you retrieve it (e.g. is it thrown when calling login before the redirect or after etc).

Also can you look at your auth0 logs and see what if there are any errors in there as well?

**Comment by jwstrivr** [comment]
![debug](https://github.com/auth0-samples/auth0-WinFormsWPF-oidc-samples/assets/150383689/84425042-8866-43bc-9cd4-34698d6e1912)
Here are the exact debug properties I receive from the endpoint

From the auth0 logs, I see only success confirmations for the request. From our own log streams, it looks like the requested scope is missing which seems to be consistent with the error since that requires the `openid` scope. There are no rules that exist to modify the access token ingestion either

**Comment by jwstrivr** [comment]
@frederikprijck any update on this? Since AuthZ Extension is being deprecated sometime fall of next year I'll need to move my WPF application to use Auth0 Core which seems to be impossible without NuGet package support from Auth0

**Comment by jwstrivr** [comment]
Correct, I'm picking organizations under Applications > My WPF App under the Organizations tab where I see Types of Users. If I select Individuals, the application works fine, but when I move to Business Users + Prompt for Credentials I am able to select the organization from the Auth0 login flow but receive the error in the screenshot above. Do I need to include organizations somewhere within the code?

Just to be clear, the Individuals option is the one that will be deprecated within the year, correct?

**Comment by frederikprijck** [comment]
> Do I need to include organizations somewhere within the code?

No you shouldnt need to.

> Just to be clear, the Individuals option is the one that will be deprecated within the year, correct?

Can you share the information about the deprecation you are talking about?

**Comment by jwstrivr** [comment]
Here's a link to the article:

https://auth0.com/blog/preparing-for-rules-and-hooks-end-of-life/?utm_medium=email&utm_source=mkto&utm_campaign=2023-05-email-rules-hooks-deprecation&mkt_tok=ODU1LVFBSC02OTkAAAGLxPDEUtB4yr0NQmmHDFLxJ_d8CdorUFgWKDCi3zo5kc3n-9xiUgavkvCIr1M8eeFHMA99tiLkcJeUlALNjErQwP5YgA3PaH7Eh530t6Cc1DPKU9_i
